docs/Basic-Guide.md

```diff
@@ -216,22 +207,22 @@ INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 10000. Mean Reward: 2
 ### After training
 
 You can press Ctrl+C to stop the training, and your trained model will be at
-`models/<run-identifier>/<brain_name>.nn` where
-`<brain_name>` is the name of the Brain corresponding to the model.
+`models/<run-identifier>/<behavior_name>.nn` where
+`<behavior_name>` is the name of the `Behavior Name` of the agents corresponding to the model.
```

Comment:
"agent's"

Reply:
I think 'agents' is meant to be plural plural here

Reply:
oops, you're right.